src/Components/CustomNode.jsx

```diff
@@ -0,0 +1,254 @@
+import React, { memo, useState } from "react";
+import { Handle, Position } from "reactflow";
+import "../Styles/CustomNode.css";
+// import Switch from "../Styles/Switch";
+import "../Styles/Options.css";
+export default memo(({ data, isConnectable }) => {
+  let mainblockClassName = "mainblock"; // Default class
+
+  // different color blocks places
+  if (data.label === "Client send request") {
+    mainblockClassName += " clientColorBlock";
```

Comment:
nit: turn these strings into enum strings
something like this
```
enum Classnames {
    Color = "serverColorBlock",
    ......
}
```

Reply:
Did something similar to. Good suggestion